cmd/exec.go

```diff
@@ -59,7 +58,6 @@ var execTaskCmd = &cobra.Command{
 		gpuEnabled, _ := cmd.Flags().GetBool(gpuEnabledFlag)
 		if gpuEnabled {
 			logger.Info().Msgf("starting task w/ gpu enabled")
-			executable = fmt.Sprintf("LD_PRELOAD=%s %s", utils.GpuSharedLibPath, executable)
```

Comment:
I think we should leave this in so customers don't have to manually LD_PRELOAD their gpu tasks. Does this not work on your machine @yashanand1910 ?

Reply:
Oh! This was simply moved to the daemon. Check `process.go`, where we start the controller. Just thought to keep it at one place.

Reply:
Ah missed that - thank you! Yeah better in process than cmd/exec.